cypress/integration/tests/controls/stakeholders/crud.test.ts

```diff
@@ -20,15 +20,15 @@ describe("Stakeholder CRUD operations", () => {
         cy.intercept("PUT", "/api/controls/stakeholder/*").as("putStakeholder");
     });
 
-    it("Stakeholder crud operations", function () {
-        const stakeholder = new Stakeholders(data.getStakeholderEmail(), data.getStakeholderName());
+    it("Stakeholder CRUD operations", function () {
+        const stakeholder = new Stakeholders(data.getEmail(), data.getFullName());
         // Create new stakeholder
         stakeholder.create();
         cy.wait("@postStakeholder");
-        stakeholder.exists();
+        cy.get(tdTag).should("contain", stakeholder.email);
```

Comment:
i think exists doing same except clicking on stakeholder tab and selecting items per page.

Reply:
But that has extra wait time plus it goes back to click on control tab and click back again on stakeholder page tab. Which causes extra time taking for test. To be very, concise, the exist and not exist methods are mere one line of assertion, which don't need to wrapped in function.

Reply:
Modified the exists and notExist functions and used the same in tests.